packages/components/src/filters/date/content.js

```diff
@@ -91,7 +92,7 @@ class DatePickerContent extends Component {
 										after={ after }
 										before={ before }
 										onUpdate={ onUpdate }
-										invalidDays="future"
+										isInvalidDate={ dateString => moment().isBefore( moment( dateString ), 'day' ) }
```

Comment:
Optional: What would you think about abstracting these one liners out to functions, so the name can make them a bit clearer (i.e. this one would be futureDaysInvalid or something)? I liked that 'future' previously implied that any future days were invalid. Now it takes a second to parse what's going on. This one is also used twice.